include/pfs/drp/stella/math/Math.h

```diff
@@ -583,6 +583,13 @@ namespace pfs { namespace drp { namespace stella {
                                      ndarray::Array< U, 2, 1 > const& valuesIfTrue,
                                      ndarray::Array< U, 2, 1 > const& valuesIfFalse );
     
+    template< typename T, typename U >
+    ndarray::Array< U, 2, 1 > where( ndarray::Array< T, 2, 1 > const& arrayToCompareTo,
+                                     std::string const& op,
+                                     T const valueToCompareTo, 
+                                     ndarray::Array< U, 2, 1 > const& valuesIfTrue,
+                                     ndarray::Array< U, 2, 1 > const& valuesIfFalse );
+    
```

Comment:
Good point that it matches the declaration before. I guess something went wrong with a rebase. I removed this commit.
I filed a ticket [PIPE2D-156](https://pfspipe.ipmu.jp/jira/browse/PIPE2D-156) to remove all `where` function declarations from `stellaLib.i` as `numpy.where` should be used instead.
Regarding the speed in C++, in the majority of use cases the arrays `valuesIfTrue` and `valuesIfFalse` exist already, so they won't have to be created just to call the `where` function